cypress/support/pages/teams/pageCommonTeams.js

```diff
@@ -20,110 +19,129 @@ class Teams_Common {
   static #selectInternalTeamMember = '[data-testid="select_team_members_add"]'
   static #studentTableBody = '[data-testid="students_names_container"]'
   static #confirmTeamMemberAddButton = '[data-testid="btn-submit"]'
-  static #confirmTeamMemberDeleteButton= '[data-testid="btn-submit"]'
+  static #confirmTeamMemberDeleteButton = '[data-testid="btn-submit"]'
   static #deleteIconInTableViewRow = '[data-testid="btn-delete-team-member"]'
 
-
-  doNotSeeDeletedStudentInTeam(){
-    cy.get(Teams_Common.#studentTableBody).find('tr td','Herbert').should('not.exist')
+  doNotSeeDeletedStudentInTeam () {
+    cy.get(Teams_Common.#studentTableBody)
+      .find('tr td', 'Herbert')
+      .should('not.exist')
   }
 
   removeStudentInTeam () {
-    cy.get(Teams_Common.#studentTableBody).contains('tr', 'Herbert').then(tableRow => {
-      cy.wrap(tableRow).find(Teams_Common.#deleteIconInTableViewRow).click()
-      cy.get(Teams_Common.#confirmTeamMemberDeleteButton).contains('Teilnehmer:in löschen').click()
-    })
+    cy.get(Teams_Common.#studentTableBody)
+      .contains('tr', 'Herbert')
+      .then(tableRow => {
+        cy.wrap(tableRow)
+          .find(Teams_Common.#deleteIconInTableViewRow)
+          .click()
+        cy.get(Teams_Common.#confirmTeamMemberDeleteButton)
+          .contains('Teilnehmer:in löschen')
```

Comment:
Isn't there an id we can use instead of label.
Currently we change the language to german after each login, but if we change this some day, this line will not work anymore.